packages/yourloops/pages/hcp/team-members-table.tsx

```diff
@@ -261,34 +245,44 @@ function MembersTableBody(props: TeamMembersProps): JSX.Element {
         data-role={member.role}
         data-status={member.status}
       >
-        <TableCell id={`team-members-list-${team.id}-row-${userId}-icon`}>{icon}</TableCell>
+        <TableCell id={`team-members-list-${team.id}-row-${userId}-icon`}>
+          {member.status === UserInvitationStatus.pending &&
+            <Tooltip title={t("pending-invitation") as string} aria-label={t("pending-invitation")} placement="bottom">
```

Comment:
I don't understand what is the relation between this code and e-cps certified icon ...